ext/bcmath/libbcmath/src/str2num.c

```diff
@@ -49,85 +48,96 @@ bool bc_str2num(bc_num *num, char *str, size_t scale)
 	/* Check for valid number and count digits. */
 	ptr = str;
 
-	if ((*ptr == '+') || (*ptr == '-')) {
-		/* Skip Sign */
-		ptr++;
-	}
+	ptr += (*ptr == '-' || *ptr == '+');
```

Comment:
This is true, but adding a comment can't help (i.e. if we have a sign we need to increment the str pointer)